### PR TITLE
fix bad rendering of the changelog for the alias grammar changes

### PR DESCRIPTION
**Pull request by ghost**
So the grammar block is [not well rendered](https://dlang.org/changelog/pending.html#aliasdecly_func), I don't know why. Never mind, I remove it because the changelog has never exposed a $(GRAMMAR ) block to this date. Just a link will be fine for those who are interested.

@wilzbach, you still need to enable markdown preview when the change log is rendered. I tried to do it but the 1000 lines makefile of dlang.org repo causes me allergic reactions.

### CODE REVIEW

**Comment by wilzbach** [comment]
> @wilzbach, you still need to enable markdown preview when the change log is rendered. 

IIRC @dgileadi was/is on this.

**Comment by Basile-z** [comment]
Sorry I've seen a review comment that let me think that you would do it.

@dgileadi, can you enable it ? This changelog entry uses it too  now.

File: changelog/aliasdecly_func.dd [hunk 1]
```diff
@@ -1,23 +1,10 @@
 A new syntax is available to declare an alias to a function type
 
 It's now possible to declare an alias to a function type using the `alias` syntax based on the assignment operator.
-The declaration consists of the return type, followed by parameters and ends with optional (member) function attributes:
+The declaration consists of the return type, followed by parameters and ends with optional (member) function attributes. See
+[AliasDeclarationY](https://dlang.org/spec/grammar.html#AliasDeclarationY) for more details.
```
Comment (wilzbach) [inline changelog/aliasdecly_func.dd]: This is Ddoc, no Markdown. You need to use e.g. `LINK2` or other macros.
I'm afraid enabling -preview=markdown won't happen overnight as it probably will result in a few changes.

Comment (Basile-z) [reply changelog/aliasdecly_func.dd]: yes but dgileadi also uses markdown in his dd entry.

Comment (Basile-z) [reply changelog/aliasdecly_func.dd]: [clearly](https://github.com/dlang/dmd/blob/master/changelog/ddoc_markdown.dd#L3)

Comment (wilzbach) [reply changelog/aliasdecly_func.dd]: Have a look at the preview: https://dlang.org/changelog/pending.html#ddoc_markdown

Also, I pointed it out on his PR too: https://github.com/dlang/dmd/pull/9645

Comment (Basile-z) [reply changelog/aliasdecly_func.dd]: Yes, this is where i understood that you'd put the new switch in the makefile. Anyway i've replaced with a LINK2 now, no need to bother with this details. It's just that he has to fix his link too.

**Comment by Basile-z** [comment]
@thewilsonator merge now ? Just rebased and squashed.

**Pull request merged**